playground/Makefile

```diff
@@ -21,6 +21,9 @@ WHEEL_NAME := spylang-$(VERSION)-py3-none-any.whl
 # Build playground for local development
 # This mirrors the steps from .github/workflows/playground.yml
 local:
+	@echo "Cleaning libspy..."
+	$(MAKE) -C $(LIBSPY_DIR) clean
+
```

Comment:
I don't fully understand why a simple `make` is not enough.
In theory, `make` should automatically rebuild when necessary: if this doesn't happen, maybe it means that we should fix libspy makefile?

Note that I'm happy add the `make clean` as a temporary workaround if needed, but I'd like to understand why it's needed (and possibly insert a comment to explain)